src/main/resources/META-INF/configurations/org.jahia.modules.api.authorization-default.yml

```diff
@@ -32,7 +43,7 @@ site_tree:
 html_views:
   description: Can access HTML views
   auto_apply:
-    - origin: hosted
+    - always: true
```

Comment:
is it necessary to have **true** here, as we only test for not null in AlwasyAutoApply.java

`public static AutoApply build(PropertiesValues values) {
        String value = values.getProperty("always");
        if (value != null) {
            return new AlwaysAutoApply();
        }

        return null;
    }`

Reply:
I need to have a value different from null, can be anything